app/uk/gov/hmrc/hmrcemailrenderer/templates/awrs/awrsNotificationCancellationAPI10.scala.html

```diff
@@ -0,0 +1,8 @@
+@(params: Map[String, Any])
+@uk.gov.hmrc.hmrcemailrenderer.templates.helpers.html.template_main(params, "Alcohol Wholesaler Registration Scheme (AWRS) cancellation") {
```

Comment:
On these emails we don't know who we are addressing, therefore, we are unable to add a salutation.